app/components/landing/featuredProjectSection/slider/SliderPill.tsx

```diff
@@ -9,20 +9,21 @@ import Typography from '@mui/material/Typography';
 
 import * as m from '@/src/paraglide/messages.js';
 
-const PillBox = ({ children, sx }: { children: ReactNode; sx?: SxProps }) => {
+const PillBox = ({ children, sx, active }: { children: ReactNode; sx?: SxProps; active?: boolean }) => {
   const style = {
     border: '0.50px white solid',
     px: '10px',
     py: '1px',
     display: 'flex',
     alignItems: 'center',
+    minHeight: active ? '65px' : '48px',
     ...sx,
   };
   return <Box sx={style}>{children}</Box>;
 };
 
-const ActiveSliderPill = (props: { itemNumber: string; title: string; projectStart: string }) => {
-  const { itemNumber, title, projectStart } = props;
```

Comment:
@naadnn please only remove the time, not the date
![Screenshot 2024-10-15 at 17 18 22](https://github.com/user-attachments/assets/ee1eb821-cc7a-44d4-85e4-9e6a26ddc862)

Reply:
Resolved in second commit

Reply:
I think the fix requires a bit of more research. Your fix does fix the problem (on the client side) but not the cause of the problem:
- In strapi, projects do not have a start time, only a date (see the strapi [project content-type](https://github.com/openkfw/innoverse/blob/main/strapi/src/api/project/content-types/project/schema.json#L54)) -> type is **date**. 
- So the question is where does the time come from actually? it must be set somewhere, please inspect where it is set and why, it might be a bug in a feature or function.

- Hint: there are other strapi content types that do have a start time that includes a time as well, like the events ([see here](https://github.com/openkfw/innoverse/blob/main/strapi/src/api/event/content-types/event/schema.json#L66)) -> type is **datetime**